assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -86,32 +89,38 @@ const useQuickActions = () => {
     currentPage?.elements.find((element) => element.isBackground) ||
     selectedElements?.[0]?.isBackground;
 
-  const defaultActions = useMemo(
-    () => [
+  const defaultActions = useMemo(() => {
+    const actionMenuProps = {
+      tooltipPlacement: isRTL ? PLACEMENT.LEFT : PLACEMENT.RIGHT,
+    };
```

Comment:
nit: this could probably be outside of this memoization since it'll get reused on all the different action sets.